### ISSUE TITLE
Miskelly - Add Sales Comments back to Invoice

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2229]
_Imported from trac ticket 2229.
Created by **aadams**
Opened in trac: Mar 18, 2013, 9:50:13 AM
Last modified in trac: Apr 15, 2013, 10:44:33 AM_
***
_by aadams:_
At some point during the changes to the sales invoices, we lost all comments. We would like for the **“Sales”** comments to print on every invoice, but **no delivery or system comments**.

Not sure if release [#3811](3811) had something to do with this or not.

***
* aadams attached [Miskelly Sample Invoice.docx](https://trac.retailarchitects.com/trac/attachment/ticket/2229/Miskelly Sample Invoice.docx) on Mar 18, 2013, 4:13:06 PM

**Comment by kentbower** [comment]
_**sshah** trac comment on Mar 18, 2013, 11:40:54 AM:_

(Changeset [[3920]](../commit/cc421ce4b3807cb833c0a0862ab3f3ca4c74956d))
* Fixes #2229 Include delivery comments for Miskelly's invoice.

**Comment by kentbower** [comment]
_**sshah** trac status change on Mar 18, 2013, 11:40:54 AM:_ **closed**

**Comment by kentbower** [comment]
_**aadams** trac comment on Mar 18, 2013, 4:13:06 PM:_

I have attached the sample invoice that we intended to use at Go Live (I think a variation of El Dorado's?)...not sure why Kane's sample was loaded, or a variation of Kane's. Is it possible to completely change our invoices to this format?

___
From: Product Development@RetailArchitects [mailto:issues@retailarchitects.com] 
Sent: Monday, March 18, 2013 9:50 AM
Cc: dev@retailarchitects.com
Subject: [trac] #2229: Miskelly - Add Sales Comments back to Invoice

#2229: Miskelly - Add Sales Comments back to Invoice
___
             Reporter:  aadams            |       Owner:  sshah
                 Type:  defect            |      Status:  new
             Priority:  major             |   Milestone:
            Component:  multiple systems  |    Keywords:  mis
           Blocked By:                    |    Blocking:
        Referenced By:                    |  References:
         Story Points:                    |  User Story:
References User Story:                    |         WBS:
___
 At some point during the changes to the sales invoices, we lost all  comments. We would like for the **“Sales”** comments to print on every  invoice, but **no delivery or system comments**.


 Not sure if release [#3811](3811) had something to do with this or not.

--
Ticket URL: https://trac.retailarchitects.com/trac/ticket/2229
trac https://trac.retailarchitects.com/trac
RMS

[attachment:"Miskelly Sample Invoice.docx"]

**Comment by kentbower** [comment]
_**aadams** trac status change on Mar 18, 2013, 4:13:06 PM:_ **reopened**

**Comment by kentbower** [comment]
_**sjohnson** trac comment on Mar 19, 2013, 5:31:33 PM:_

Delivery comments were added back, Delivery needs to be removed and **Sales** Comments need to be added to Miskelly invoice.  Miskelly is not happy with the modifications to the standard invoice.  They want a duplicate of El Dorado's.
So need that with Credit card language, Sales Comments, WF language, and Treatment indicators to go on to Miskelly letterhead which is much less obtrusive than El Dorado's.

**Comment by kentbower** [comment]
_**sshah** trac status change on Apr 15, 2013, 10:44:33 AM:_ **closed**
